Gopkg.lock

```diff
@@ -56,6 +56,25 @@
   pruneopts = "NUT"
   revision = "de5bf2ad457846296e2031421a34e2568e304e35"
 
+[[projects]]
```

Comment:
These two entries look wrong -- I don't believe there should be any imports from the mobile-security-service-operator or unifiedpush-operator in here (and I can't see anywhere in this PR where they're used).

Reply:
Fixed with dep manager.